packages/mesh-graphql-client/src/types.ts

```diff
@@ -0,0 +1,402 @@
+import { SignedOrder } from '@0x/types';
```

Comment:
Some of the types in this file are duplicated from `packages/mesh-browser-lite/src/types.ts`. This is not a merge blocker for me, but I'd for us to try to consolidate these types eventually, unless there is a reason why you want to keep them separated.

Reply:
I want to address this separately. See https://github.com/0xProject/0x-mesh/issues/859. It's true that we might be able to use some of the types from __packages/mesh-browser-lite/src/types.ts__ here. But we can't use all of them. There are some subtle differences in how some of the types are used in the context of GraphQL.

I know that this PR is leaving us in a (temporarily) worse place in regards to type duplication and bulky conversion code, but I have a plan for how to address this across the entire codebase. Will comment on https://github.com/0xProject/0x-mesh/issues/859 to elaborate soon.

Reply:
Sounds good to me.